pkg/controllers/nodeclaim/garbagecollection/controller.go

```diff
@@ -99,6 +99,11 @@ func (c *Controller) Reconcile(ctx context.Context) (reconcile.Result, error) {
 			errs[i] = client.IgnoreNotFound(err)
 			return
 		}
+		pods := &corev1.PodList{}
+		var podCount int
+		if err := c.kubeClient.List(ctx, pods, client.MatchingFields{"spec.nodeName": node.Name}); err == nil {
```

Comment:
Should we include DS pods in this calculation or should we just stick to reschedulable pods (running pods)